docs/customizing-templates.rst

```diff
@@ -33,6 +33,35 @@ title for any of the fields by using the ``scaffold.fields`` configuration
         // The rest of the fields to display here
     ]);
 
+Adding Controller Actions to utilize Crud Actions
+-------------------------------------------------
+
+It's easy to add an action to a controller that makes use of another
+*CrudView* action.
+
+This does use the template provided by the edit action:
```

Comment:
Need to use two ":" in .rst files to get a colon.

Reply:
Not quite, but the preview in my repo looks good now, finally.
Thanks!